lxd/storage/drivers/driver_powerflex_utils.go

```diff
@@ -23,17 +22,18 @@ import (
 )
 
 // powerFlexBlockVolSuffix suffix used for block content type volumes.
-const powerFlexBlockVolSuffix = ".block"
+const powerFlexBlockVolSuffix = ".b"
 
 // powerFlexISOVolSuffix suffix used for iso content type volumes.
-const powerFlexISOVolSuffix = ".iso"
+const powerFlexISOVolSuffix = ".i"
```

Comment:
Please can you rebase this so there isnt an earlier version of this commit in the PR. 

Only the final version should be in the PR. Thanks

Reply:
Fixed.